### PR TITLE
Broadcast update

### PR DESCRIPTION
**Pull request by qttylog**
I thought a Bradcast function was a good idea. What do you think?

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -39,6 +39,11 @@
 except KeyError:
     REDIS_HOST = "localhost"
 
+try:
+    MODERATORS = os.environ["MENSTRUATION_MODERATORS"]
+    MODERATORS = list(MODERATORS.split(","))
```
Comment (kmein) [inline bot.py]: Please do this in one expression. Constants shouldn't be reassigned. I propose sth like

`MODERATORS = (os.environ["..."] or "").split(",")`

File: bot.py [hunk 2]
```diff
@@ -281,6 +286,41 @@ def status_handler(update: Update, context: CallbackContext):
     )
 
 
+def broadcast_handler(update: Update, context: CallbackContext):
+    """"For moderators only"""
+    logging.debug(f"Entering: broadcast_handler, chat_id: {update.message.chat_id}, MODERATORS: {MODERATORS}")
+    if str(update.message.chat_id) not in MODERATORS:
+        logging.warning(f"{update.message.chat_id} tried to send an Broadcast, but is no Moderator")
```
Comment (kmein) [inline bot.py]: "tried to send a broadcast message"

File: bot.py [hunk 3]
```diff
@@ -281,6 +286,41 @@ def status_handler(update: Update, context: CallbackContext):
...
+        context.bot.send_message(
+            update.message.chat_id,
+            f"Du hast nicht die Berechtigung einen Broadcast zu versenden. {emojize(error_emoji())}"
```
Comment (kmein) [inline bot.py]: `emojize(f"Du hast nicht die Berechtigung, einen Broadcast zu versenden. {error_emoji()}")`

File: bot.py [hunk 4]
```diff
@@ -281,6 +286,41 @@ def status_handler(update: Update, context: CallbackContext):
...
+        )
+        return None
+    text = demojize(" ".join(context.args))
+    if not text:
+        context.bot.send_message(
+            update.message.chat_id,
+            f"Broadcast-Text darf nicht leer sein. {emojize(error_emoji())}"
```
Comment (kmein) [inline bot.py]: Please apply emojize to the whole string, not only the emoji

File: bot.py [hunk 5]
```diff
@@ -281,6 +286,41 @@ def status_handler(update: Update, context: CallbackContext):
...
+            continue
+        logging.debug(f"Send to {user_id}, text: {text}")
+        try:
+            context.bot.send_message(user_id, emojized_text)
+        except Unauthorized:
+            logging.exception(f"Skipped {user_id}, because he blocked the bot")
+            continue
+    context.bot.send_message(
+        update.message.chat_id,
+        "Broadcast erfolgreich versendet."
```
Comment (kmein) [inline bot.py]: Maybe add a thumbs up emoji or something cool and creative here

**Review by kmein — CHANGES_REQUESTED** [review]

File: bot.py (hunk 5 above)
Comment (qttylog) [reply bot.py]: Yeah, good idea. Why am I always so boring? ^^

**Review by kmein — APPROVED** [review]

**Pull request merged**